### DISCUSSION TITLE
kpanda/user-guide/backup/install-velero/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1960] [Announcements]
# kpanda/user-guide/backup/install-velero/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/backup/install-velero/

**Comment by liyuerich** [comment]
文档上的参数和目前dce5.0界面有所不同，能否更新一下？
